modules/patient-portal/src/main/frontend/src/patient-portal/PatientAccessConfiguration.jsx

```diff
@@ -64,16 +67,29 @@ function PatientAccessConfiguration() {
     allowedPostVisitCompletionTime: ["Patients can fill out surveys after the associated event for:"],
     draftLifetime: ["Patients can edit unsubmitted responses for:", "-1 means that drafts are kept until the patient is no longer able to access their surveys"]
   };
+  const errorText = `Value must be between ${DEFAULT_PATIENT_ACCESS_CONFIG['draftLifetime']} and ${patientAccessConfig?.allowedPostVisitCompletionTime}`;
```

Comment:
`DEFAULT_PATIENT_ACCESS_CONFIG['draftLifetime']` is not necessarily the lowest allowed value. It happens to be 0 now, but if for any reason we had decided to make the default 3, we should still allow to configure more frequent cleanups. Please hardcode 0 as the lowest allowed value.

Reply:
Nevermind, I was misremembering the default we're using and its meaning. The implementation is correct, but the message should say something along the lines of:
```suggestion
  const errorText = `Please use a value between 0 and ${patientAccessConfig?.allowedPostVisitCompletionTime}, or ${DEFAULT_PATIENT_ACCESS_CONFIG['draftLifetime']} to disable periodic draft deletion.`;
```